scripts/delete_lambda_iam_role.py

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env python3
+
+import sys
+import boto3
+from azul import config
+
+
+lambda_name = sys.argv[1]
+role_name = config.qualified_resource_name(lambda_name)
+
+iam = boto3.client('iam')
+
+try:
+    response = iam.get_role(RoleName=role_name)
+    role = response['Role']
+except iam.exceptions.NoSuchEntityException:
+    role = None
+
+try:
+    policies = iam.list_role_policies(RoleName=role_name)
+except iam.exceptions.NoSuchEntityException:
+    policies = None
+
+if policies is not None:
+    for policy_name in policies['PolicyNames']:
+        try:
+            print(f"Deleting IAM Policy {policy_name}")
```

Comment:
I know the other script uses print() for historic reasons but we should use loggers instead.